lte/protos/oai/s1ap_state.proto

```diff
@@ -41,8 +52,17 @@ message EnbDescription {
   map<uint32, uint64> ue_id_map = 14; // mme_ue_s1ap_id -> comp_s1ap_id
 }
 
+enum S1apUeState {
+ S1AP_UE_INVALID_STATE = 0;
+ S1AP_UE_WAITING_CSR = 1; ///< Waiting for Initial Context Setup Response
```

Comment:
Comment needs to be updated. Should be Create session Request not ICSRsp

Reply:
From s1ap perspective and at here, CSR would stand for Context Setup Response; Yes to avoid confusion, I shall rename the state as S1AP_UE_WAITING_ICSR